### DISCUSSION TITLE
Deploying never ends with Vercel. Help... "solid-start rendering index.html..."

### DISCUSSION THREAD

**Discussion by atzzCokeK** [solidjs/solid-start#525] [Q&A]
I would like to deploy my project on Vercel, but "solid-start rendering index.html..." does't look working well, and it it never done.
After I added "ProviderComponent(Context)" on my project, it doesn't work. How I use is wrong? But when I do run locally and deploy it, there is no problem.
I check my vite.config.ts around, but seems well and normal. I don't know why and I am stuck with it for more than week.

Would you please help me?
My project: https://github.com/atzzCokeK/solid-js
About Context developed: https://github.com/atzzCokeK/solid-js/blob/master/src/root.tsx
Screenshot: 
<img width="1196" alt="image" src="https://user-images.githubusercontent.com/25091548/206645625-453f7c29-6bfb-4b8c-bae3-2c23a6fc9369.png">

**Comment by AshSimmonds** [comment]
It is known. Unknown if there's bandwidth to figure out how to fix it.

https://github.com/solidjs/solid-start/issues/383

**Reply by atzzCokeK** [reply]
Thank you, I saw the issue before, but I don't know the same reason or not. I am not sure if I make a mistake of my developing.
